Controller/Adminhtml/Imgix/Post.php

```diff
@@ -65,9 +65,10 @@ public function execute()
 
         $apiKey = $this->helperData->getSecureApiKey();
         $apiKeyValidation = $this->helperData->getImgixApiKeyValidation($apiKey);
-        if ($apiKeyValidation['authorized'] == 1) {
-            $isEnable = $this->helperData->isEnabled();
-            if ($isEnable == 1) {
+        $isEnable = $this->helperData->isEnabled();
+        
+        if ($isEnable == 1) {
+            if ($apiKeyValidation['authorized'] == 1) {
```

Comment:
What's the reasoning behind switching the order of these checks (`isImgixEnabled` and checking for `authorized`)?

Reply:
If imgix module disable from the system configuration and open the product in admin side it throw an error like
**imgix error: could not parce the Authorization header.**

So I have checked imigx module is enable before api key is authorized. Don't need to check api key validation if module is disabled.